easybuild/easyconfigs/i/intel/intel-2017b.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'Toolchain'
+
+name = 'intel'
+version = '2017b'
+
+homepage = 'http://software.intel.com/en-us/intel-cluster-toolkit-compiler/'
+description = """Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and Fortran compilers, Intel MPI &
+ Intel MKL."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+compver = '2017.4.196'
+gccver = '6.3.0'
```

Comment:
GCC 6.4.0 is imminent: https://gcc.gnu.org/ml/gcc/2017-06/msg00101.html (hat tip @ocaisa)